src/app/style.less

```diff
@@ -50,3 +50,12 @@ body {
     }
   }
 }
+
+.ant-drawer-body {
+  height: calc(100vh - @drawer-header-height);
+}
+
+// ensure modals display above drawers
+.ant-modal-mask, .ant-modal-wrap {
+  z-index: 1001;
```

Comment:
This fixes an issue where the password prompt modal would be displayed behind the drawer if any modal was once displayed since the page loaded.

![image](https://user-images.githubusercontent.com/1356600/50854076-16117680-1352-11e9-9f67-c4a3fee0dfd0.png)